zuul.d/ansible-plugin-builder-jobs.yaml

```diff
@@ -6,4 +6,9 @@
 - job:
     name: ansible-test-integration-ansible-plugin-builder
     parent: ansible-test-integration-base
-    nodeset: openvswitch-2.15.0-python38
+    nodeset: openvswitch-2.9.0-python38
+    host-vars:
+      openvswitch-2.9.0:
```

Comment:
Rename it `controller`.

Reply:
The nodeset is defined here: https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.sf.d/nodesets.yaml#L152-L160